scripts/update-apk.sh

```diff
@@ -29,8 +29,8 @@ fi
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
     echo "Push to master branch detected, signing the app..."
     cp app-release-unsigned.apk app-release-unaligned.apk
-    jarsigner -verbose -tsa http://timestamp.comodoca.com/rfc3161 -sigalg SHA1withRSA -digestalg SHA1 -keystore ../scripts/key.jks -storepass $STORE_PASS -keypass $KEY_PASS app-release-unaligned.apk $ALIAS
-    ${ANDROID_HOME}/build-tools/26.0.3/zipalign -v -p 4 app-release-unaligned.apk app-release.apk
+    jarsigner -tsa http://timestamp.comodoca.com/rfc3161 -sigalg SHA1withRSA -digestalg SHA1 -keystore ../scripts/key.jks -storepass $STORE_PASS -keypass $KEY_PASS app-release-unaligned.apk $ALIAS
+	${ANDROID_HOME}/build-tools/26.0.3/zipalign -p 4 app-release-unaligned.apk app-release.apk
```

Comment:
I think 26.0.3 should replace with `$BUILD_TOOLS_VERSION` to solve the `directory not found` Travis error.

Reply:
@liveHarshit ok. But I do not understand how it makes a difference.

Reply:
Build successful, build tool version is the same :+1:

Reply:
I think the build tool version is different, in Travis script it is  `ANDROID_BUILD_TOOLS_VERSION=25.0.3`

Reply:
Just test it by running the full script, and exit before the release commands.

Reply:
@liveHarshit sorry for the delay in response. Thanks for the help. I'll try updating the `ANDROID_BUILD_TOOLS_VERSION` to `28.0.3` and then running the script.